pyk/src/pyk/kast.py

```diff
@@ -81,7 +81,16 @@ class KAtt(KAst, Mapping[str, Any]):
     atts: FrozenDict[str, Any]
 
     def __init__(self, atts: Mapping[str, Any] = {}):
-        object.__setattr__(self, 'atts', FrozenDict(atts))
+
+        def _freeze(m: Any) -> Any:
+            # TODO: This should properly recurse through the data structure.
+            if isinstance(m, dict):
+                return FrozenDict(((k, _freeze(v)) for (k, v) in m.items()))
+            return m
+
+        frozen = _freeze(atts)
+        assert isinstance(frozen, FrozenDict)
+        object.__setattr__(self, 'atts', frozen)
```

Comment:
I didn't want to pull in the full complexity for `sanitize_config` into `KProve.parse_token`. This meant that some Attrs are left in the Ast. In the case of variables, the `org.kframework.kore.Sort` attribute is a dictionary. This causes a crash as dicts are not hashable (and worse, are mutable). This code converts those to FrozenDicts.

I think it's probably a good idea to keep these attributes around too, since they encode the sort information for variables.